scalableminds/webknossos
# backend unit tests for nml parser and writer

- backend now has unit-tests for nml parser and writer
- they work similar to the frontend tests

### Steps to test:
- call `sbt "test-only *NMLUnitTestSuite"` in wK directory to see the tests running

### Issues:
- fixes #2826 

------
